functions/src/users/index.js

```diff
@@ -1,45 +1,64 @@
 const admin = require("firebase-admin");
-const serviceAccount = require("../../credentials.json");
+// const serviceAccount = require("../../credentials.json");
+const serviceAccount = require("../../cred.json");
```

Comment:
same thing here... please name your file consistently to mine.

Reply:
done